### DISCUSSION TITLE
Develop new protocol for varying inverter

### DISCUSSION THREAD

**Discussion by paulsteigel** [ArminJo/JK-BMSToPylontechCAN#12] [Ideas]
Hi Armin,
I tried the CAN bus using the pylontech CAN protocol, however, the Luxpower SNA5000 can not recognize the data (LuxPower announce that it can recognize pylontech battery with battery type of 2, lion battery).
I came across this: https://us.v-cdn.net/6034073/uploads/OJ8IDWBXR179/seplos-bms-communication-protocol-v2-0.pdf. The seplos BMS can talk to Luxpower Inverter via CAN bus.
Therefore, I am thinking of proposing you to follow the data structure provided in this document to allow the modification be recognized by Luxpower Inverter. Do you think this can be feasible?
Please advice with thanks
Ngoc

**Comment by ArminJo** [comment]
Done 😀

**Reply by paulsteigel** [reply]
Many thanks for your great works!
Ngoc!
